automatic/apache-httpd/tools/helpers.ps1

```diff
@@ -4,8 +4,8 @@ function Assert-TcpPortIsOpen {
         [Parameter(Position = 0, Mandatory, ValueFromPipeline)][ValidateNotNullOrEmpty()][int] $portNumber
     )
 
-    $process = Get-NetTCPConnection -State Listen -LocalPort $portNumber -ErrorAction SilentlyContinue | `
-        Select-Object -First 1 -ExpandProperty OwningProcess | `
+    $process = ( netstat -aon | foreach { if ( $_ -match ":$portNumber" ) { $_ } } )-split " " | `
```

Comment:
`netstat -aon | ? {$_ -match ... }` or `(netstat -aon) -match ...`

Reply:
Sounds like a good suggestion. I'll change it now